### DISCUSSION TITLE
1.3.0

### DISCUSSION THREAD

**Discussion by rtivital** [mantinedev/mantine#109] [Releases]
## Release demos

[June 2021 release demos](https://mantine.dev/pages/changelog/)

## New components, hooks and features

- New [RingProgress](https://mantine.dev/core/ring-progress/) component – display circular progress with multiple sections
- [Tabs](https://mantine.dev/core/tabs/#outline-variant) component now support outline variant
- [Badge](https://mantine.dev/core/badge/#colors) component now supports new dot variant
- [NumberInput](https://mantine.dev/core/number-input/#custom-incrementdecrement-controls) component now exposes increment/decrement handlers via ref 
- New [Center](https://mantine.dev/core/center/) component – simple wrapper to center element horizontally and vertically
- New [use-interval](https://mantine.dev/hooks/use-interval/) hook provides simple wrapper around `window.setInterval`
- [use-focus-trap](https://mantine.dev/hooks/use-focus-trap/#initial-focus) hook now allows to change element that will initially receive focus

## Project core changes

- [New documentation design](https://mantine.dev) with tabs and less cluttered packages index pages
- All packages are now published in 3 formats: esm (ECMAScript modules `export { Button } from './Button';`), cjs (commonjs `export.Button = Button;`) and umd (default for browser). These changes will allow webpack and similar bundlers to perform better tree shacking.
- All components now export related types: `import { ButtonProps } from '@mantine/core';`

## Design changes

- [Button](https://mantine.dev/core/button/) component label is no longer uppercased by default, to replicate old design use `<Button uppercase />`
- Colors saturation and text shadows adjusted for [Button](https://mantine.dev/core/button/) and [Badge](https://mantine.dev/core/badge/) components for better contrast

<hr /><em>This discussion was created from the release <a href='https://github.com/mantinedev/mantine/releases/tag/1.3.0'>1.3.0</a>.</em>

**Comment by rtivital** [comment]
Special thanks:

- @cstrat for providing lots of feedback
- @blackcater for refactoring mantine build scripts 
- @khushal87 for working on RingProgress component and Badge dot variant

Thank you! Your contributions are really valuable to the project

**Reply by cstrat** [reply]
Fantastic update - thanks @rtivital !

@khushal87 I love the dot variant on the badge. Nice work!

**Reply by khushal87** [reply]
Thanks @cstrat :confetti_ball:
